custom_model_runner/datarobot_drum/drum/push.py

```diff
@@ -164,6 +165,55 @@ def print_model_started_dialogue(new_model_id):
     )
 
 
+def validate_training(config, options):
+    # Setting default values for most of these :)
+    path = Path(config["validation"]["input"])
+    if not os.path.isabs(path):
+        path = Path(options.code_dir).joinpath(path)
+
+    options.input = path
+    options.output = None
+    options.negative_class_label = None
+    options.positive_class_label = None
+    options.target_csv = None
+    options.target = config["validation"]["targetName"]
+    options.row_weights = None
+    options.row_weights_csv = None
+    options.num_rows = "ALL"
+    options.skip_predict = False
+
+    raw_args_for_docker = "drum fit --input {input} --target {target} --code-dir {code_dir}".format(
+        input=path, target=options.target, code_dir=options.code_dir
+    ).split()
+
+    return options, RunMode.FIT, raw_args_for_docker
+
+
+def validate_inference(config, options):
+    path = Path(config["validation"]["input"])
+    if not os.path.isabs(path):
+        path = Path(options.code_dir).joinpath(path)
+
+    options.input = path
+    options.output = "/dev/null"
+    options.negative_class_label = None
+    options.positive_class_label = None
+    raw_args_for_docker = "drum validate --input {input} -cd {code_dir}".format(
```

Comment:
How binary classification is validated?

Reply:
The predictions of the outputs are never checked, the data is just fed in.